docs/FAQ.md

```diff
@@ -314,3 +314,10 @@ Please see [these docs](https://docs.victoriametrics.com/vmctl.html#migrating-da
 ## How to migrate data from Graphite to VictoriaMetrics?
 
 Please use [whisper-to-graphite](https://github.com/bzed/whisper-to-graphite) tool for reading the data from Graphite and pushing it to VictoriaMetrics via [Graphite import API](https://docs.victoriametrics.com/#how-to-send-data-from-graphite-compatible-agents-such-as-statsd).
+
+
+## Why do same metrics have differences in VictoriaMetrics and Prometheus dashboards?
```

Comment:
The explanation is not full. There are two aspects to consider:
1. Difference in stored raw time series. Due to different compressions algorithms, VM may reduce precision for float values with more than 12 significant decimal digits. Please, read more about this here https://valyala.medium.com/evaluating-performance-and-correctness-victoriametrics-response-e27315627e87
2. Difference in query responses. Please read more here https://medium.com/@romanhavronenko/victoriametrics-promql-compliance-d4318203f51e